spec/rubocop/cop/chef/deprecation/hwrp_without_resource_name_and_provides_spec.rb

```diff
@@ -73,27 +73,9 @@ class UlimitRule < Chef::Resource
     RUBY
   end
 
-  # This is correct usage for >= Chef-16, but this will fail on <= Chef-15 (this should become the autocorrect case in >= Chef-16)
-  # This is allowed here under the assumption that the person is writing cookbooks for >= Chef-16
-  it 'registers an offense when a Resource derived HWRP does not define a resource_name' do
-    expect_no_offenses(<<~RUBY)
-    class Chef
-      class Resource
-        class UlimitRule < Chef::Resource
-          provides :ulimit_rule
```

Comment:
yeah you probably shouldn't delete this.  we want to allow this.

Reply:
At the moment we're enforcing a Chef 15 compatible syntax which requires that be there.